core/sections/clause_3_references.adoc

```diff
@@ -0,0 +1,31 @@
+
+[bibliography]
+== References
+
+* [[[OGC_14-065,OGC 14-065]]], OGC 14-065, OGC WPS 2.0 Interface Standard, version 2.0.2
+
+* [[[OGC_06-121r9,OGC 06-121r9]]], OGC 06-121r9, OGC Web Service Common Specification, version 2.0
+
+* [[[OGC_08-131r3,OGC 08-131r3]]], OGC 08-131r3 -- The Specification Model -- A Standard for Modular Specifications
+
+* [[[jsonschemavalidation,JSON Schema Validation]]] JSON Schema Validation: A Vocabulary for Structural Validation of JSON. https://json-schema.org/draft/2020-12/json-schema-validation.html
```

Comment:
OGC references order: OGC, ISO, other SDOs, then non-standards. Move to end.

Reply:
Done.